go.mod

```diff
@@ -4,16 +4,10 @@ go 1.22.0
 
 retract v1.3.0
 
-replace github.com/GoogleContainerTools/kpt-functions-sdk/go/fn => github.com/GoogleContainerTools/kpt-functions-sdk/go/fn v0.0.0-20220506190241-f85503febd54
-
 require (
 	cloud.google.com/go/iam v1.1.12
 	github.com/GoogleContainerTools/kpt v1.0.0-beta.48
-	github.com/GoogleContainerTools/kpt-functions-catalog/functions/go/apply-replacements v0.1.1
-	github.com/GoogleContainerTools/kpt-functions-catalog/functions/go/apply-setters v0.2.0
-	github.com/GoogleContainerTools/kpt-functions-catalog/functions/go/set-namespace v0.4.1
-	github.com/GoogleContainerTools/kpt-functions-catalog/functions/go/starlark v0.4.3
-	github.com/GoogleContainerTools/kpt-functions-sdk/go/fn v0.0.0-20220506190241-f85503febd54
+	github.com/GoogleContainerTools/kpt-functions-sdk/go/fn v0.0.0-20230427202446-3255accc518d
```

Comment:
How do we handle this one going forward also?
This is being used in the nephio codebase too.

Reply:
now both our code and the 3rd party KRM function code relies on the same version of the sdk, namely the latest.